src/UI/IMDbRating.js

```diff
@@ -20,15 +20,8 @@ const StyledIcon = styled.img`
 const IMDbRating = (props) => {
   return (
     <StyledDiv>
-      <StyledIcon
-        src={IMDbIcon}
-        alt="oops"
-        width="auto"
-        height={props.iconHeight}
-      />
-      <StyledRating height={props.textHeight}>
-        {props.rating.toFixed(1)}/10
-      </StyledRating>
+      <StyledIcon src={IMDbIcon} alt="oops" width="auto" height="20px" />
```

Comment:
why are you hard coding the heaight ? also the font size, they should probably be passed as props.